client/extensions/hello-dolly/state/selectors.js

```diff
@@ -0,0 +1,19 @@
+
+/**
+ * External Dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import lyrics from './lyrics';
+
+export const getCurrentLyric = ( lines ) => ( reduxState ) => {
```

Comment:
Minor: why not just call the variable `state`? We're in a file of path `state/selectors.js`, so I don't think there will be doubts as to whether we're talking about Redux state or React component state.